examples/language/openmoe/benchmark/benchmark_cai.py

```diff
@@ -264,6 +301,9 @@ def main():
             optimizer.step()
             optimizer.zero_grad()
             performance_evaluator.on_step_end(exmaple_data["input_ids"])
+            if (step == args.warmup // 2) and args.load_balance:
```

Comment:
Is this the standard method to use load balance?

Reply:
it is just for benchmark. it will be updated to train later. we will check the load every n steps and do update if load is not balanced